cypress/e2e/claim-portal-failure.cy.ts

```diff
@@ -0,0 +1,267 @@
+    statusCode: 200,
+    body: {},
+  });
+}
+
+function stubEthereum(address?: string, signer?: JsonRpcSigner) {
+  cy.on("window:before:load", (win) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    ((win as any).ethereum = {
+      isMetaMask: true,
+      enable: cy.stub().resolves([address]),
+      request: cy.stub().callsFake(async (method) => {
+        if (method == "eth_requestAccounts") {
+          return [address];
+        }
+        if (method == "wallet_sendDomainMetadata") {
+          return true;
+        }
+        if (method == "wallet_addEthereumChain") {
+          return true;
+        }
+        if (method == "wallet_switchEthereumChain") {
+          return true;
+        }
+        if (method == "wallet_watchAsset") {
+          return true;
+        }
+        if (method == "eth_chainId") {
+          return "0x7a69";
+        }
+        if (method == "eth_accounts") {
+          return [address];
+        }
+        if (method == "eth_signTypedData_v4") {
+          return "address";
+        }
+
+        if (method == "eth_estimateGas") {
+          return "0x00";
+        }
+
+        if (method == "eth_sendTransaction") {
+          return "0x";
+        }
+
+        if (method == "eth_call") {
+          return "0x";
+        }
+      }),
+      on: cy.stub().callsFake((event, cb) => {
+        if (event == "accountsChanged") {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (win as any).ethereum.onAccountsChanged = cb;
+        }
+      }),
+      autoRefreshOnNetworkChange: false,
+      chainId: "0x7a69",
+      selectedAddress: address,
+      requestAccounts: cy.stub().resolves([address]),
+      send: cy.stub().callsFake(async (method) => {
+        if (method == "eth_requestAccounts") {
+          return [address];
+        }
+        if (method == "wallet_sendDomainMetadata") {
+          return true;
+        }
+        if (method == "wallet_addEthereumChain") {
+          return true;
+        }
+        if (method == "wallet_switchEthereumChain") {
+          return true;
+        }
+        if (method == "wallet_watchAsset") {
+          return true;
+        }
+        if (method == "eth_chainId") {
+          return "0x7a69";
+        }
+        if (method == "eth_accounts") {
+          return [address];
+        }
+        if (method == "eth_signTypedData_v4") {
+          return;
+        }
+
+        if (method == "eth_sendTransaction") {
+          return "0x";
+        }
+
+        if (method == "eth_call") {
+          return "0x";
+        }
+
+        if (method == "eth_estimateGas") {
+          return "0x00";
+        }
+
+        if (method == "wallet_watchAsset") {
+          return true;
+        }
+      }),
+    }),
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      signer ? ((win as any).signer = signer) : null;
+  });
+}
+
+// placed here due to length
+const claimUrl =
```

Comment:
This permit has a value of `0.01`, is it ok? Wouldn't we prefer a value of `0`?

Reply:
Not particularly no, `0` would still be a valid amount as far as I'm aware in terms of will `0` still produce a tx to sign, I believe it would because the permit itself is still valid and any incorrect values would through the UI in a different direction. But I will try it out nonetheless